### PR TITLE
Update incompatible_mods.txt

### PR DESCRIPTION
**Pull request by aubergine10**
Removed `Supply Chain Coloring` and `Transfer Broker BETA` as those don't seem to contain any malicious code.

Chaos cloned TM:PE; added it to the list. https://steamcommunity.com/sharedfiles/filedetails/?id=2749420751

This is _not_ out of spite, we already list all TM:PE clones as incompatible in the file, including the obsolete version I hosted (LABS). I've just found a couple more and added those too.

```
1957033250;Traffic Manager: President Edition (Industries Compatible)
1806963141; TM:PE LABS
1546870472;Traffic Manager: President Edition (Industries Compatible)
1581695572;Traffic Manager: President Edition
1348361731;Traffic Manager: President Edition ALPHA/DEBUG
498363759;Traffic Manager + Improved AI
563720449;Traffic Manager + Improved AI (Japanese Ver.)
492391912;Improved AI (Traffic++)
409184143;Traffic++
626024868;Traffic++ V2
568443446;Traffic Manager Plus 1.2.0
481786333;Traffic Manager Plus
427585724;Traffic Manager
```

That's just the ones from the top of the file.

### CODE REVIEW

File: TLM/TLM/Resources/incompatible_mods.txt [hunk 1]
```diff
@@ -79,4 +79,5 @@
 2730687809;Network Extensions 3 - contains malware targeting TM:PE devs
 2399343344;Harmony (redesigned) - by same author of the malware, and potentially much more dangerous
 2389228470;Transfer Broker BETA - by same author of the malware
-2325122732;Supply Chain Coloring - by same author of the malware
\ No newline at end of file
+2325122732;Supply Chain Coloring - by same author of the malware
```
Comment (kvakvs) [inline TLM/TLM/Resources/incompatible_mods.txt:82]: Will TM:PE compat checker print this message? Put full message in each line
Imagine it will be printing "by same author" without context?

Comment (aubergine10) [reply TLM/TLM/Resources/incompatible_mods.txt:82]: No, I think it pulls the mod name from the mod; those comments in the file are just to make it easier for us to maintain the list of workshop ids. I'll check though just to be sure.

Comment (aubergine10) [reply TLM/TLM/Resources/incompatible_mods.txt:82]: Ok, checked and the displayed text is always based on the plugin info / mod name (the text in the .txt file is never shown to user): 
image

**Review by kvakvs — APPROVED** [review]

File: TLM/TLM/Resources/incompatible_mods.txt (hunk 1 above)
Comment (DaEgi01) [reply TLM/TLM/Resources/incompatible_mods.txt:82]: I'd still be rather professional about it ;D

Comment (aubergine10) [reply TLM/TLM/Resources/incompatible_mods.txt:82]: It's not out of spite; if user has two TM:PE mods subbed at same time, even if one is disabled, it can cause huge problems with save/load of mod otions and road customisations due to duplicate assembly bug (Mono loads both TMPEs due to different build number in the assembly info).

Comment (aubergine10) [reply TLM/TLM/Resources/incompatible_mods.txt:82]: I think we can remove the two older chaos mods as they are locked from updates and don't seem to contain anything untoward (that I can find anyway).

I'll send additional commit to remove them from list.

Comment (DaEgi01) [reply TLM/TLM/Resources/incompatible_mods.txt]: I'm just talking about the "- by same author of the malware".
Incompatible mods should of course be marked, no matter what.
But it would be more helpful to write "- due to Patching method xyz" or "- due to broken functionality reported by x" or something like that.

**Comment by aubergine10** [comment]
Found some more TM:PE clones and added to list; need to test in app to make sure there aren't any duplicate id's in the list.

**Review by krzychu124 — APPROVED** [review]

**Comment by aubergine10** [comment]
@DaEgi01 oops sorry, didn't spot your review request - lmk if there's any changes you want
